backend/variant_search/views.py

```diff
@@ -11,29 +12,23 @@ class VariantViewSet(viewsets.ModelViewSet):
     # Apply and then filter by the gene param if set
     queryset = Variant.objects.exclude(gene='')
     serializer_class = VariantSerializer
+    filter_backends = (filters.SearchFilter,)
```

Comment:
hmm, I think we'd purposely left the query / filtering stuff out, but were candidates finding issues with the current setup?

Reply:
When I get into pair programming with candidates, one of the things I look for is whether they are using built-in capabilities of the stack we've given them to work with. Even if someone isn't terribly familiar with DRF, by leveraging those capabilities in the example app, I think it's a clue to them that these exercises can use other built-in capabilities. For example, the entirety of the backend changes needed for the sorting exercise can be implemented by changing this one line to:
`filter_backends = (filters.SearchFilter, filters.OrderingFilter)`

Reply:
As far as that comment that "we'd purposely left the query / filtering stuff out", the existing example does implement searching, so nothing has changed as far as the basic endpoint support for searching by gene. This change only moves the implementation of that searching out of the endpoint, by leveraging `filters.SearchFilter`.